pkg/asset/agent/manifests/common.go

```diff
@@ -1,62 +1,78 @@
 package manifests
 
-// import (
-// 	"github.com/openshift/installer/pkg/asset/agent"
-// 	"github.com/openshift/installer/pkg/asset/installconfig"
-// 	"github.com/openshift/installer/pkg/types"
-// 	corev1 "k8s.io/api/core/v1"
-// 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-// 	"k8s.io/utils/pointer"
-// )
+import (
```

Comment:
If this file is meant to store shared test utils, for clarity I'd suggest to add the `_test.go` extension. If it's not possible due the imports, then please add a `test` suffix in the file name, ie `agenttest`.